config/hardening/ssg-supplemental.sh

```diff
@@ -294,52 +292,58 @@ cat <<EOF > /etc/audit/rules.d/audit.rules
 -w /var/log/btmp -p wa -k session
 -w /var/log/wtmp -p wa -k session
 
+#2.6.2.4.12 Ensure auditd Collects System Administrator Actions
+-w /etc/sudoers -p wa -k actions
+
+#2.6.2.4.13 Make the auditd Configuration Immutable
+-w /sbin/insmod -p x -k modules
+-w /sbin/rmmod -p x -k modules
+-w /sbin/modprobe -p x -k modules
+-a always,exit -F arch=b32 -S init_module -S delete_module -k modules
+-a always,exit -F arch=b64 -S init_module -S delete_module -k modules
+
+# Ignore all the anonymous events. Often tagged on every rule, but ignoring 
+# up front should improve processing time
+-a exit,never -F auid=4294967295
+# Ignore system services
+-a exit,never -F auid<1000
```

Comment:
@shawndwells Yes, I agree they were removed from the individual rules. Lines 307 and 309 were added that ignore everything where auid<1000 or auid=-1
Since 307/309 are above the rules where the constraint was removed, those rules will never be reached for system daemons.